### PR TITLE
Fixing native file export #1158

### PR DESCRIPTION
**Pull request by erangatl**
## Purpose
Fixing native file export #1158

### CODE REVIEW

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/js/dialog/export-file-dialog.js [hunk 1]
```diff
@@ -29,252 +29,27 @@ define(['require', 'lodash', 'jquery', 'log', 'backbone', 'file_browser', 'boots
...
+                var downloadLink = document.createElement("a");
+                    downloadLink.download = title;
+                    downloadLink.innerHTML = "export File";
+                    downloadLink.href = window.URL.createObjectURL(payload);
+                    downloadLink.onclick = destroyClickedElement;
+                    downloadLink.style.display = "none";
+                    document.body.appendChild(downloadLink);
+                    window.URL.revokeObjectURL(payload);
+                    downloadLink.click();
+              function destroyClickedElement(event) {
```
Comment (dnwick) [inline components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/js/dialog/export-file-dialog.js]: @erangatl please format the code to indent properly

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/js/dialog/export-file-dialog.js [hunk 2]
```diff
@@ -29,251 +29,27 @@ define(['require', 'lodash', 'jquery', 'log', 'backbone', 'file_browser', 'boots
...
-                            callback(false);
-                        }
-                    });
+                var activeTab = app.tabController.activeTab;
+                var siddhiFileEditor= activeTab.getSiddhiFileEditor();
+                var config = siddhiFileEditor.getContent();
+                var payload = new Blob([config], {type: "text/plain; charset=utf-8"});
+                var downloadLink = document.createElement("a");
+                    downloadLink.download = title;
+                    downloadLink.innerHTML = "export File";
```
Comment (dnwick) [inline components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/js/dialog/export-file-dialog.js]: if possible lets move these to constants

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/js/dialog/export-file-dialog.js (hunk 1 above)
Comment (erangatl) [reply components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/js/dialog/export-file-dialog.js]: Fixed with #1690

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/js/dialog/export-file-dialog.js (hunk 2 above)
Comment (erangatl) [reply components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/js/dialog/export-file-dialog.js]: Fixed with #1690

**Pull request merged**